docs/index.rst

```diff
@@ -1,14 +1,69 @@
-.. DISPATCHES documentation master file, created by
-   sphinx-quickstart on Thu Jan  7 17:58:50 2021.
-   You can adapt this file completely to your liking, but it should at least
-   contain the root `toctree` directive.
+.. index::
+   pair: dispatches;Home
 
-Welcome to DISPATCHES' documentation!
-=====================================
+DISPATCHES
+==========
+The **D**\ esign **I**\ ntegration and **S**\ ynthesis **P**\ latform to **A**\ dvance **T**\ ightly
+**C**\ oupled **H**\ ybrid **E**\ nergy **S**\ ystems (DISPATCHES) project, is being developed and
+used to identify and optimize Integrated Energy Systems for operation within the bulk power system
+via energy market signals.
 
+DISPATCHES is part of the DOE Grid Modernization Laboratory Consortium (GMLC).
+
```

Comment:
I think you should add links to the github repo and home page.

Reply:
I added a link for DISPATCHES to the github repo, but what home page do you mean?

Reply:
ok, I linked to the github.io site and the GMLC main page here.